### DISCUSSION TITLE
Ledger Live DApp Browser wagmi connector

### DISCUSSION THREAD

**Discussion by chabroA** [wevm/wagmi#676] [Show and tell]
Hello 🙌 

We have created a [ledger-live-wagmi-connector](https://www.npmjs.com/package/@ledgerhq/ledger-live-wagmi-connector) to make integrating DApps in the Ledger Live wallet easier 👷 

To learn more about how and why integrating your project in the Ledger Live platform, have a look at our [Developer Portal](https://developers.ledger.com/docs/live-app/start-here/)

Feel free to give us feedback! ❤️

**Comment by tmm** [comment]
Added to [`awesome-wagmi`](https://github.com/tmm/awesome-wagmi#connectors=)!

**Reply by chabroA** [reply]
Thanks! 😃
